.travis.yml

```diff
@@ -24,6 +30,28 @@ before_install:
   - mkdir -p ${HOME}/.m2
   - cp settings.xml $HOME/.m2
 
+  - export PREVIEW_NAMESPACE=$(echo "v$TRAVIS_BRANCH-$TRAVIS_BUILD_NUMBER"|  tr '[:upper:]' '[:lower:]'|tr . -)
+  - ORG='activiti'
+  - export HELM_RELEASE_NAME="$TRAVIS_BRANCH-$TRAVIS_BUILD_NUMBER"
+  - export GLOBAL_GATEWAY_DOMAIN="35.242.205.159.nip.io"
```

Comment:
GLOBAL_GATEWAY_DOMAIN should be travis env variable

Reply:
done